### ISSUE TITLE
Restricted Equipment

### ISSUE THREAD

**Issue by michaelphagen** [dwmorrin/gas-eco#105]
Requested by Michael McCoy. Some equipment should be restricted in terms of who can check it out. Specifically, only students who have completed ETR should be able to check out the Tube Mics and the Coles. Ideally, the solution for this would automatically keep track of whether students have completed ETR (possibly by checking how long they have existed in RMSS, or whether they took ETR via RMSS), but a manual flag solution could easily work for now.

**Comment by dwmorrin** [comment]
As I mentioned to Michael McCoy in the meeting, this may have to continue to be a human check at this point.

As it stands, I do not believe there is an algorithm we can give the computer with the data we have to determine eligibility.  We can apply some heuristics, but by the time we've made a messy and less than perfect system, I doubt we've improved much on just having human judgement do the work.
